processor/attributesprocessor/README.md

```diff
@@ -56,6 +59,22 @@ For the `hash` action,
   action: hash
 ```
 
+
+For the `extract` action,
+ - `key` is required and will
+ ```yaml
+ # Key specifies the attribute to extract values from.
+ # The value of `key` is NOT altered.
+- key: <key>
+  # Rule specifies the regex pattern used to extract attributes from the value
+  # of `key`.
+  # The submatchers must be named.
+  # If attributes already exist, they will be overwritten.
+  pattern: <regular pattern with named matchers>
+  action: extract
```

Comment:
I assume this will work similarly to how spanProcessor `to_attributes` setting works except it will use an existing attribute instead of span name as the source?

Reply:
Yes it will behave similar to the span processor to_attributes just with an existing attribute as the source. I'll add that as a comment.